webapp/components/integrations/components/add_command.jsx

```diff
@@ -84,7 +84,8 @@ export default class AddCommand extends React.Component {
             method: this.state.method,
             username: this.state.username,
             icon_url: this.state.iconUrl,
-            auto_complete: this.state.autocomplete
+            auto_complete: this.state.autocomplete,
+            team_id: this.props.team.id
```

Comment:
They're needed as of the v4 API, which no longer gets the team id from a path parameter. Without these changes, adding and editing new or existing commands is completely broken. But they were merged in a separate PR (https://github.com/mattermost/platform/pull/6799), so when I make the other requested changes, these will disappear from this review.